apps/web/src/pages/TalentNominations/NominationGroup/components/CurrentPositionExperiences.tsx

```diff
@@ -110,6 +157,20 @@ const CurrentPositionExperiences = ({
             data-h2-flex-direction="base(column)"
             data-h2-gap="base(x.5 0)"
           >
+            {sorted.length === 0 && (
+              <>
+                <p
```

Comment:
This wasn't in the AC so not blocking, but we usually put these types of messages in a well.  I'd suggest a "warning" colour, but somethign like this:
![image](https://github.com/user-attachments/assets/1d6fdb97-1316-416c-8120-12c13de9aff6)

Reply:
Reminder to use tailwind classes for anything new. I've made the commit: [30d352f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13776/commits/30d352f4fcc8b40b5c624d630801afa414772b9b)